src/sentry/services/hybrid_cloud/user/__init__.py

```diff
@@ -196,6 +252,22 @@ def serialize_user(cls, user: User) -> APIUser:
             roles = frozenset(flatten(user.roles))
         args["roles"] = roles
 
+        options: FrozenSet[APIUserOption] = frozenset({})
+        if hasattr(user, "options") and user.options is not None:
+            options = frozenset(
+                [
+                    APIUserOption(
+                        id=o["id"],
+                        project_id=o["project_id"],
+                        organization_id=o["organization_id"],
+                        key=o["key"],
+                        value=dbsafe_decode(o["value"]),
```

Comment:
Me again :) This line + the changes to `issue_occurrence.py` are causing an issue when we try to use the endpoint. Once decoded, `value` is (not sure if this is always or just sometimes) a dictionary and causes a TypeError. I'm not sure how to want to flatten / store it.